intelmq/lib/upgrades.py

```diff
@@ -952,6 +953,27 @@ def v322_removed_feeds_and_bots(configuration, harmonization, dry_run, **kwargs)
     return '\n'.join(messages) if messages else None, configuration, harmonization
 
 
+def v340_deprecations(configuration, harmonization, dry_run, **kwargs):
+    """
+    Rename twitter parser, warn on Twitter collector
+    """
+    changed = None
+    found_twitter_collector = []
+    message = None
+    for bot_id, bot in configuration.items():
+        if bot_id == 'global':
+            continue
+        if bot["module"] == "intelmq.bots.parsers.twitter.parser":
+            configuration[bot_id]["module"] = "intelmq.bots.parsers.ioc_extractor.parser"
+            changed = True
+        elif bot["module"] == "intelmq.bots.collectors.twitter.collector":
+            found_twitter_collector.append(bot_id)
+
+    if found_twitter_collector:
+        message = f"Found discontinued Twitter collector bot: {', '.join(found_twitter_collector)}"
```

Comment:
same comment. The user will see this (hopefully) and then ... what?

Reply:
Right, there's the instruction missing. Will add it.

Reply:
fixed